eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,8 @@ $PackageRepository = "Nuget"
 $packagePattern = "*.nupkg"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/main/_data/releases/latest/dotnet-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=dotnet%2F&delimiter=%2F"
+$GithubUri = "https://github.com/Azure/azure-sdk-for-net"
+$PackageRepositoryUri = "https://www.nuget.org/packages"
```

Comment:
Are these used by scripts in `eng/common`?

Reply:
Yes. It is here:https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Helpers/Service-Level-Readme-Automation-Helpers.ps1#L49